src/socket.rs

```diff
@@ -1017,6 +1119,28 @@ impl Drop for Socket {
     }
 }
 
+/// Fix-up a created socket by setting the proper portability flags.
```

Comment:
These flags aren't for portability. These flags are avoiding a possible resource leak (inherit/clos-exec) and unhandled signals (sigpipe).

Reply:
Well, I'd consider not having to worry about these platform specific things makes it portable, but fine.